.circleci/scripts/verify-docs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Verifies that docs have been generated and updated
+GREEN='\e[0;32m'
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+md5Command=md5sum
+if [ $(uname -s) == "Darwin" ]; then
+    md5Command=(md5 -q)
+fi
+
+# Store MD5 of Committed Documentation
+MD5_ROOT_README_BEFORE=$(find README.md -type f -exec $md5Command {} \; | sort -k 2 | $md5Command)
+MD5_DOCS_BEFORE=$(find ./documentation -type f -exec $md5Command {} \; | sort -k 2 | $md5Command)
+
+# Update Table of Contents
+./.circleci/scripts/update-tocs.sh
+# Update Codox documentation
+lein docs
+
+# Calculate Later
+MD5_ROOT_README_AFTER=$(find README.md -type f -exec $md5Command {} \; | sort -k 2 | $md5Command)
+MD5_DOCS_AFTER=$(find ./documentation -type f -exec $md5Command {} \; | sort -k 2 | $md5Command)
+
+# Verify root README.md
+if [ "$MD5_ROOT_README_BEFORE" != "$MD5_ROOT_README_AFTER" ]; then
+    printf "${RED}Aborting, parent README file TOC was not updated${NC}\n"
```

Comment:
I do not follow this: when `"$MD5_ROOT_README_BEFORE" != "$MD5_ROOT_README_AFTER"`  is true it means README was updated, is not it? Message says otherwise.

Reply:
So there are automated scripts that pass through the README file to generate a table of contents, likewise our HTML documentation is generated via `lein docs`.

These scripts take a hash of these locations and then re-run the generating commands, if the hashes don't match it means that the contributor has changed something (added a new header in the readme, a new function, etc) and forgot to re-generate and commit the files.

It's not comparing the file from master and the active branch, which is what I think you are thinking of.

Reply:
Ah, makes sense.